sdk/core/Microsoft.Extensions.Azure/README.md

```diff
@@ -0,0 +1,54 @@
+# Azure client integration for ASP.NET Core
+
+Microsoft.Extensions.Azure.Core provides shared primitives to integrate Azure clients with ASP.NET Core dependency injection and configuration systems.
+
+[Source code][source_root] | [Package (NuGet)][package]
+
+## Installing
+
+Install the ASP.NET Core integration library using [NuGet][nuget]:
+
+```
+dotnet add package Microsoft.Extensions.Azure
+```
+
+## Usage Scenarios and Samples
+
+Use add `AddAzureClients` call to you `ConfigureServices` method. You can use provided builder to register client instances with dependency injection container.
```

Comment:
Nit - "Make a call to `AddAzureClients` in your app's `ConfigureServices` method.  You can use the provided builder to register client instances with your dependency injection container."